src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -37,7 +37,12 @@ export const UserAvatar: React.FC<UserAvatarProps> = ({
     "user-avatar--clickable": onClick !== undefined,
   });
 
+  const status = user?.data?.status || "";
```

Comment:
Any reason we want to have "" as a default? Why not just having it undefined?

Reply:
(Also we default with ?? not ||)

Reply:
I think that status shouldn't be `undefined` in any case. Just my opinion.

Reply:
@oleksii-front Why do you say it shouldn't be `undefined` if we have no other value for it? TypeScript makes use of `undefined` all over the place, it's a better value than an arbitrary 'empty string' or other placeholder value as we can easily explicitly check against it and react accordingly.

All of the `?.` 'safe accessor' stuff is neatly wrapping potentially `undefined` cases. We use it a lot in our codebase.

- https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining